site/en/blog/google-summer-of-code-and-chrome-extensions/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Google Summer of Code & Chrome Extensions
+description: >
+  The experience and advice on participating in Google Summer of Code.
+date: 2023-08-29
+authors:
+  - xuezhoudai
+
+hero: 'image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/8ZNtuWAIcc7NBdS3AQEv.jpg'
+alt: >
+  Google Summer of Code & Chrome Extensions
+
+sharing_image: 'image/wVNVUJS8Z8O04i1tJKSdsp6nkRQ2/8ZNtuWAIcc7NBdS3AQEv.jpg'
+---
+
+I'm a sophomore from China passionate about web development. In my first year, I joined a technical club at our college. This club was my introduction to coding and open source. In the club, I met a group of like-minded partners who love coding. And it was from them that I learned about Google Summer of Code at the beginning of this year. This Google-organized global program connects students with open-source organizations and guides them in making good use of their summer by participating in open-source activities.
+
+With the attitude of having a try, I decided to submit my application. Fortunately, I was accepted. The summer I spent contributing to the Chrome Extension Samples repo has been memorable and precious. Of course, I have also learned a lot: communicating effectively, coding skills, and planning abilities, among other things.
```

Comment:
```suggestion
Thinking, "why not?", I submitted my application. Fortunately, I was accepted. The summer I spent contributing to the Chrome Extension Samples repo has been memorable and precious. Of course, I've also learned a lot: effective communication, coding skills, and planning abilities, among other things.
```

Reply:
I think the first part is more like "giving it a try". IMO The rest seems okay to leave the original text.

Reply:
Done.